source/server/server.h

```diff
@@ -46,12 +46,12 @@ namespace Server {
  * All server wide stats. @see stats_macros.h
  */
 // clang-format off
-#define ALL_SERVER_STATS(COUNTER, GAUGE)                                                           \
+#define ALL_SERVER_STATS(BOOL_INDICATOR, COUNTER, GAUGE)                                           \
   GAUGE(uptime)                                                                                    \
   GAUGE(concurrency)                                                                               \
   GAUGE(memory_allocated)                                                                          \
   GAUGE(memory_heap_size)                                                                          \
-  GAUGE(live)                                                                                      \
+  BOOL_INDICATOR(live)                                                                             \
```

Comment:
probably move this to the first line of the macro, alphabetizing first by stat-type and then by stat-name.

Reply:
Done.